test/integration/crud/bulk.test.ts

```diff
@@ -1688,6 +1688,55 @@ describe('Bulk', function () {
     }
   });
 
+  it('should apply hint via FindOperators', {
+    metadata: { requires: { mongodb: '>= 4.2' } },
```

Comment:
I see failures on 4.2 for this, should it just be "greater than"?

Reply:
I was going by the version number mentioned on https://www.mongodb.com/docs/manual/reference/method/Bulk.find.hint/, but didn’t realize that at least for deletes at least 4.3.4 is required. I’m good with just bumping the entire test to `>= 4.4`, but also happy to split it if that’s better (there are also spec tests for `bulkWrite()` that cover the different server version ranges).

Reply:
I'm happy with 4.4+ coverage